css/css-text/text-align/text-align-match-parent-002.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Test: text-align - computed value for match-parent on the root element</title>
+<link rel="help" href="https://drafts.csswg.org/css-text/#valdef-text-align-match-parent">
+<meta name="assert" content="'text-align: match-parent' on the root element with 'direction: rtl' should compute to 'right'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  html {
+    text-align: match-parent;
+    direction: rtl;
+  }
+  #log {
+    direction: ltr;
+    text-align: start;
+  }
+</style>
+<div id="log"></div>
+<script>
+  test(() => {
+    assert_equals(getComputedStyle(document.documentElement).textAlign, 'right');
```

Comment:
The spec has now been modified (see https://github.com/w3c/csswg-drafts/issues/6542), making the test as written incorrect. It should now be updated to say:
```suggestion
    assert_equals(getComputedStyle(document.documentElement).textAlign, 'start');
```